src/main/java/mate/academy/hibernate/relations/Main.java

```diff
@@ -16,17 +22,17 @@ public static void main(String[] args) {
         SessionFactory sessionFactory = HibernateUtil.getSessionFactory();
```

Comment:
Creating SessionFactory in every class where you use it can lead to performance issues. Consider using Dependency Injection or Singleton pattern to avoid this problem.